src/Microsoft.Extensions.PlatformAbstractions/PlatformServices.cs

```diff
@@ -3,25 +3,18 @@
 
 namespace Microsoft.Extensions.PlatformAbstractions
 {
-    public abstract class PlatformServices
+    public class PlatformServices
     {
-        private static PlatformServices _defaultPlatformServices = new DefaultPlatformServices();
-
-        public static PlatformServices Default
+        private PlatformServices()
         {
-            get
-            {
-                return _defaultPlatformServices;
-            }
+            Application = new ApplicationEnvironment();
+            Runtime = new RuntimeEnvironment();
         }
 
-        public abstract IApplicationEnvironment Application { get; }
+        public static PlatformServices Default { get; } = new PlatformServices();
 
-        public abstract IRuntimeEnvironment Runtime { get; }
-        
-        public static void SetDefault(PlatformServices defaultPlatformServices)
```

Comment:
Do we want to keep this around? It's kinda useless now that you can no longer create `IApplicationEnvironment`, \ `IRuntimeEnvironment` to make this useful.